reviews/templates/reviews/review_update_email_checkbox.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base/fetc_form_base.html" %}
+
+{% load static %}
+{% load i18n %}
+
+{% block header_title %}
+    {% trans "Formulieren aanpassen" %}
+{% endblock %}
+
```

Comment:
```suggestion
{% block pre-form-text %}
    <h2>{% trans "E-mail waarschuwing toevoegen" %}</h2>
    <p>
        {% blocktrans trimmed %}
            Op deze pagina kan een waarschuwing worden toegevoegd aan de detailpagina van 
            een aanvraag, om reviewers duidelijk te maken dat er aanvullende informatie 
            is verstuurd via e-mail.
        {% endblocktrans %}
    </p>
{% endblock %}
```
I think it would be nice to add some text to this page. You can translate it as:
"On this page, you can add a warning to the detail page of a application, to indicate to reviewers that additional information has been sent via e-mail."